cmake/GNU.cmake

```diff
@@ -40,6 +39,9 @@ endif()
 if(AVX2)
     set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS}")
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
+elseif(SIMDMULTIARCH)
+    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS}")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
```

Comment:
What the the purpose of this?
A=A?

Reply:
I didn't write this code.

Reply:
Understood.
Since there is cleaning e.g. `AVX2` etc. and adding flags, it is appropriate to comment.

Reply:
This is an addition. GREEN.